experimental/packages/exporter-trace-otlp-proto/test/traceHelper.ts

```diff
@@ -238,26 +238,24 @@ export function ensureExportTraceServiceRequestIsSet(
     'resourceSpans is missing'
   );
 
-  const resource = resourceSpans[0].resource;
+  const resource = resourceSpans?.[0].resource;
   assert.strictEqual(!!resource, true, 'resource is missing');
 
-  const instrumentationLibrarySpans =
-    resourceSpans[0].instrumentationLibrarySpans;
+  const instrumentationLibrarySpans = resourceSpans?.[0].scopeSpans;
```

Comment:
rename to `scopeSpans`?

Reply:
Fixed in [e591ad3](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/e591ad38b8f178ff1bbd62da51bef185df1e8638). Thanks for catching this one! :slightly_smiling_face: